### PR TITLE
Multiple user tables so you need to specify which one you want

### PR DESCRIPTION
**Pull request by rajjanorkar**
You can now have multiple user tables so you need to specify which one you want.

### CODE REVIEW

**Comment by willvincent** [comment]
As submit this would break the package for older versions of laravel. I think we ought to avoid that. If you'd like to update your pull request so that this isn't a breaking change I'll be happy to merge it.

File: src/Rateable/Rating.php [hunk 1]
```diff
@@ -27,6 +27,6 @@ public function rateable()
      */
     public function user()
     {
-        return $this->belongsTo(Config::get('auth.model'));
+        return $this->belongsTo(Config::get('auth.providers.users.model'));
```
Comment (willvincent) [inline src/Rateable/Rating.php]: This change will break on older versions of laravel.  Off the top of my head, maybe we should instead do something like this?

```
public function user()
{
  if (!$userClass = Config::get('auth.model')) {
    $userClass = Config::get('auth.providers.users.model'));
  }

  return $this->belongsTo($userClass);
}
```

Comment (willvincent) [inline src/Rateable/Rating.php]: Or maybe it would be better to expose a config file where this can be defined

**Comment by rajjanorkar** [comment]
Thanks Will,

Yeah you are right, it will break the older versions of laravel.

I implemented it in 5.3 so thought create pull request for that version.
But your suggested code can handle old as well new version.

Thank you.
Raj

**Comment by willvincent** [comment]
Closing as this has sat untouched for 4 months. Please feel free to reopen, or open a new PR with a backward compatible solution.

**Pull request closed without merging**